packages/backend/package.json

```diff
@@ -12,9 +12,30 @@
     "directory": "packages/backend"
   },
   "license": "MIT",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    },
+    "./errors": {
+      "import": {
+        "types": "./dist/errors.d.ts",
+        "default": "./dist/errors.mjs"
+      },
+      "require": {
+        "types": "./dist/errors.d.ts",
+        "default": "./dist/errors.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
   "main": "./dist/index.js",
```

Comment:
Let's drop `main` but keep `types` since TS configs with `moduleResolution` set to `node` will not be able to resolve the types otherwise. 

TS will use the `exports` field only if `moduleResolution` is set to `node16`, `nodenext` or `bundler`. Many projects still use `node` though, so let's not be disruptive.

Extra context: https://www.typescriptlang.org/tsconfig#moduleResolution

Reply:
https://github.com/clerk/javascript/pull/2365